Gruntfile.js

```diff
@@ -0,0 +1,71 @@
+module.exports = function (grunt) {
+
+  grunt.loadNpmTasks('grunt-babel');
+  grunt.loadNpmTasks('grunt-contrib-clean');
+  grunt.loadNpmTasks('grunt-contrib-uglify');
+  grunt.loadNpmTasks('grunt-banner');
+
+  var LICENSE = '/****\n * ' +
+    grunt.file.read('./LICENSE.md', {encoding: 'utf-8'}).replace(/\n/ig, '\n * ') +
+    '\n ****/';
+
+  grunt.initConfig({
+    babel: {
+      options: {
+        ast: false,
+        sourceMaps: false,
+        nonStandard: false,
+        compact: 'false',
+        modules: 'umd',
+        moduleId: 'Timsort',
```

Comment:
The module id only affects how it behaves when used with an AMD module loader in the browser or where it'll be placed in the global object if AMD isn't available, usually `window`. So you'd reference it by `define("myModule", ["Timsort"], function(...){});` or `window.Timsort`

I just felt like the capitalization fit it a little better, but I understand what you mean.